src/main/java/com/synopsys/integration/alert/common/descriptor/config/context/ChannelDistributionDescriptorActionApi.java

```diff
@@ -86,4 +98,39 @@ public FieldModel deleteConfig(final FieldModel fieldModel) {
                    .findFirst()
                    .map(providerDescriptor -> providerDescriptor.getActionApi(ConfigContextEnum.DISTRIBUTION).orElse(null));
     }
+
+    @Override
+    public Map<String, String> validateCreate(final FieldModel fieldModel) {
+        final String jobNameError = validateJobNameUnique(fieldModel);
+        if (StringUtils.isNotBlank(jobNameError)) {
+            return Map.of(ChannelDistributionUIConfig.KEY_NAME, jobNameError);
+        }
+        return Map.of();
+    }
+
+    private String validateJobNameUnique(final FieldModel fieldModel) {
+        final String descriptorName = fieldModel.getDescriptorName();
+        return fieldModel.getField(ChannelDistributionUIConfig.KEY_NAME).map(jobNameField -> {
```

Comment:
This is a pretty heavily loaded lambda. Can we break it up a little more for readability? Maybe an additional method or two.